airbyte-webapp/src/packages/cloud/cloudRoutes.tsx

```diff
@@ -162,16 +163,18 @@ export const Routing: React.FC = () => {
 
   return (
     <WorkspaceServiceProvider>
-      <Suspense fallback={<LoadingPage />}>
-        {/* Allow email verification no matter whether the user is logged in or not */}
-        <Routes>
-          <Route path={CloudRoutes.FirebaseAction} element={<VerifyEmailAction />} />
-        </Routes>
-        {/* Show the login screen if the user is not logged in */}
-        {!user && <Auth />}
-        {/* Allow all regular routes if the user is logged in */}
-        {user && <MainViewRoutes />}
-      </Suspense>
+      <LDExperimentServiceProvider>
```

Comment:
:information_source: I'm adding this here, and not in the `App.tsx` file with the other services, so it does not load before we actually have our auth. That prevents us to unnecessarily init LaunchDarkly while our authentication is still loading and then needing to reidentify (and loading new flags) after we have the real authentication object.

Reply:
Yes, we're only running them for Cloud (for now). We're not using any ExperimentProvider to wrap our OSS page at all, thus the `useExperiment` will not get a service from `useContext` and thus return the defaultValue passed in immediately.